NEWS.md

```diff
@@ -1,5 +1,6 @@
 # mlr3torch dev
 
+* Added `CallbackSetTB`, which allows logging that can be viewed by TensorBoard.
```

Comment:
Should the most recent news be at the top?

Reply:
Usually I add them at the bottokm